float32 x_from_map
float32 y_from_map
---
bool is_possible_see

